cmake/CMakeLists.txt

```diff
@@ -743,6 +744,14 @@ if (onnxruntime_USE_TENSORRT)
   endif()
 endif()
 
+if (onnxruntime_USE_MIGRAPHX)
+  if (WIN32)
+    message(FATAL_ERROR "MIGraphX does not support build in Windows!")
```

Comment:
are there plans to eventually support in Windows?

Reply:
We currently dont have any plans.